maestrowf/interfaces/script/slurmscriptadapter.py

```diff
@@ -149,6 +136,17 @@ def get_header(self, step):
         if exclusive:
             modified_header.append("#SBATCH --exclusive")
 
+        # If neither Procs nor Nodes exist, throw an error
+        procs = resources.get("procs")
+        nodes = resources.get("nodes")
+        if procs == None and nodes == None:
+            err_msg = "At least one of either Procs or Nodes needs to be "
+            "present for Script to proceed."
+            LOGGER.error(err_msg)
+            rt_err_msg = "No explicit resources specified. At least one of " 
+            "Nodes or Procs must be set to a non-zero value."
```

Comment:
The message is fine but should be made clearer actually. When we talked earlier, I forgot to mention that we can use the step's entries to write a specific error message. In this case, make sure that nodes and procs are capitalized exactly as they are in the specification. You can also use the step to print the name of the step that's missing these entries so that the user knows exactly where to look.